### DISCUSSION TITLE
Claude.AI by Anthropic Integration

### DISCUSSION THREAD

**Discussion by kentxvalentines** [end-4/dots-hyprland#696] [Ideas]
Claude.AI is one of the top model competing hand-in-hand with ChatGPT. 
I tried to use user-options file to integrate it into the API but the issue is Anthropic uses a different syntax for passing API keys and parsing output. 

Example, Anthropic API key is passed using this header
`--header "x-api-key: YOUR_API_KEY"` instead of `Authorization: Bearer $OPENAI_API_KEY`

Even with modification to properly setup the correct API key syntax, the output is also not parsed properly like below. 
![image](https://github.com/user-attachments/assets/a758576f-14d6-45b5-80d2-b747011cf834)

I currently have an active subscription for Claude 3.5 Sonnet because of how powerful it is. It would be very helpful if there are integration into the left API panel.

**Comment by end-4** [comment]
As I am not interested in Claude, don't have a subscription, and this is going to require quite some work, I'm not interested in implementing this. Sorry.
